.travis.yml

```diff
@@ -1,9 +1,20 @@
 language: node_js
-node_js:
```

Comment:
This will run 4 build simultaneously. (2 versions * the 2 builds that get opened for every PR)

Without specifying, it will use the version in the nvmrc. I've set the number of concurrent builds to 1, so sauce won't error out because 2 builds are running simultaneously.